test/e2e/common/apparmor.go

```diff
@@ -84,7 +84,7 @@ done`, testCmd)
 		Spec: api.PodSpec{
 			Containers: []api.Container{{
 				Name:    "test",
-				Image:   "gcr.io/google_containers/busybox:1.24",
+				Image:   busyboxImage,
```

Comment:
I also have a similar opinion, will update the tests to call `GetE2EImage` function